content/posts/irods-development-update-may-2022.md

```diff
@@ -0,0 +1,286 @@
+Title: iRODS Development Update: May 2022
+Date: 2022-05-13 17:00
+Author: Kory Draughn
+Slug: irods-development-update-may-2022
+Status: published
+
+
+Hello again! Time for another monthly development update.
+
+iRODS 4.3.0 is very close. Delay server migration is now possible. With that comes new `iadmin` subcommands for setting the delay server and determining who is running the delay server. We've successfully ported the legacy Kerberos authentication plugin to the new authentication plugin framework. Issues with compilation of Python byte-code and RPM package generation have been resolved as well. Who would've guessed that was a thing.
+
+The Development Environment is receiving some enhancements in the form of debuggers and runner images for Ubuntu 20, Debian 11, and Almalinux 8. And since we're talking about Docker-based tools, you'll be happy to know that support for SSL is being added to the Testing Environment's federation test infrastructure.
+
+Finishing touches are being applied to the Indexing plugin. Once released, it will support `iput --metadata` and user/group ownership metadata. Not only that but, configuration settings that take integers as strings will also be accepted as actual integers. We know passing integers around as strings is a little weird. We hope this change improves the experience a little.
```

Comment:
"Not only that but, configuration settings that take integers as strings will also be accepted as actual integers. We know passing integers around as strings is a little weird. We hope this change improves the experience a little."

is a bit confusing/awkward - consider...

"Not only that, but the handling of configuration settings is now a bit more flexible regarding integers and strings."

Reply:
Updated. Please review.